modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbStorageEngine.java

```diff
@@ -125,31 +124,26 @@ public String name() {
 
     @Override
     public void start() throws StorageException {
-        registerDataRegion(DEFAULT_DATA_REGION_NAME);
-
+        // TODO: IGNITE-21386 Fix runtime loading for storage profiles
```

Comment:
seems like we need to remove this todos all over the PR

Reply:
Done